language/move-vm/transactional-tests/tests/entry_points/all_possible_param_types.exp

```diff
@@ -1 +1,10 @@
 processed 1 task
+
+task 0 'run'. lines 1-5:
+Error: Script execution failed with VMError: {
+    major_status: FAILED_TO_DESERIALIZE_ARGUMENT,
```

Comment:
This looks wrong I think

Reply:
It is wrong. Fixing it

Reply:
ah I see. This test in particular want strong typing suffix.